File: 0020/Classwork/classwork4.py
num = int(input("Enter any whole number: "))  #აქ მომხარებელს ვატანინებთ რიცხვსnum1 = int(input("Enter any intenger number, but bigger then previous one: "))   #აქ მომხმარებელს ვატანინებთ მეორე რიცხვს
num1 = int(input("Enter any whole number, but bigger then previous one: "))  #აქ მომხმარებელს ვატანინებთ მეორე რიცხვს
multiple = 1   #ან ვქმნით ნამრავლს
for i in range(num, num1):   #აქ ვწერთ num დან num1 ის ჩათვლით რიცხვებს
    multiple = multiple * i  #აქ ვწერთ მომხმარებლის მიერ ორი რიხცვების შორის ნამრავლს
    print(multiple)   #ხოლო აქ კი ნამრავლს გამოვსახავთ ეკრანზე

File: Day 95/Classwork/classwork4.js
/*6)შექმენი ფუქნცია რომელსაც გადასცემ რიცხვების სიას,ოღონდ ეს რიცხვები უ ნდა იყოს სტრრინგ ტიპის,შენი დავალებაა რომ იპოვო თითეულ რიცხვის ციფრების ჯამი და დააბრუნო სია სადაც იქნება თითეულიი რიცხვის ციფრების ჯამი */

function num(list) {
    let new_list = []
    for(let i = 0;i < list.length;i++) {
        let sum = 0
        for(let index = 0;index < list[i].length;index++) {
            sum += Number(list[i][index])
        }
        new_list.push(sum)
    }
    return new_list
}

console.log(num(["566", "678", "98"]))